crates/dekaf/Cargo.toml

```diff
@@ -27,6 +27,7 @@ axum = { workspace = true }
 axum-extra = { workspace = true }
 axum-server = { workspace = true }
 base64 = { workspace = true }
+humantime = { workspace = true }
```

Comment:
Given that we use Go's ParseDuration in all other places I know of, maybe we should use https://docs.rs/go-parse-duration/latest/go_parse_duration/

Thoughts? @williamhbaker @jgraettinger @psFried

Reply:
We've been using humantime for a while for this, I think we should stick with it